helm/bootzooka/templates/configmap.yaml

```diff
@@ -20,3 +21,4 @@ data:
   OTEL_EXPORTER_OTLP_PROTOCOL: "{{ .Values.bootzooka.otel.protocol }}"
   OTEL_SERVICE_NAME: "{{ .Values.bootzooka.otel.service_name }}"
   OTEL_METRIC_EXPORT_INTERVAL: "{{ .Values.bootzooka.otel.metric_export_interval }}"
+  LOGBACK_JSON_ENCODE: "{{ .Values.bootzooka.logback_json_encode }}"
```

Comment:
this doesn't seem to be used anywhere in the code, are you sure this does something? ;)

Reply:
bump

Reply:
@paapug this got merged, but is this setting needed?